Dockerfile

```
@@ -1,3 +1,9 @@
+# Build Go binary without cgo dependencies
+FROM golang:1.12.1 as builder
```

```suggestion
FROM golang:1.12 as builder
```

autobumping batch versions carries risk, but it's worth it for patch versions IMO